### PR TITLE
note for wip

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
@qsomazzi what do you think?

**Comment by qsomazzi** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Sonata Dashboard Bundle
 
 [![Build Status](https://secure.travis-ci.org/sonata-project/SonataDashboardBundle.png)](https://secure.travis-ci.org/#!/sonata-project/SonataDashboardBundle)
 
+**NOTE: This bundle ist under active development and has no stable release yet!**
```
Comment (qsomazzi) [inline README.md]: `is* under`

**Comment by OskarStark** [comment]
good catch, thank you

**Pull request merged**